content/blog/2017/2017-04-10-jenkins-has-upgraded-to-java-8.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Starting with 2.54, Jenkins now requires Java 8"
+tags:
+- java8
+author: batmat
+---
+
+We link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[announced in January]
+that Jenkins would be upgrading its Java runtime dependency to Java 8 this
+year. After a sizable amount of preparation, this week's release of
+link:/changelog/#v2.54[Jenkins **2.54**] is the first weekly release to require
+a Java 8 runtime.
+
+For users of the _weekly_ release, this means that Jenkins 2.54 **must** have
+a Java 8 runtime installed on the system in order to
+run. Those using the
+link:https://hub.docker.com/r/jenkinsci/jenkins[jenkinsci/jenkins:latest]
+Docker container won't need to take any action, as the Java runtime environment
```

Comment:
I think it's safe to assume that the container contains the right environment, since it says the user won't need to take any action